packages/ember/.npmignore

```diff
@@ -0,0 +1,32 @@
+# compiled output
```

Comment:
Kind of nitpicking but all our other `npmignore` files look something like:

```
*
!/dist/**/*
!/esm/**/*
```

which is far easier to understand and also less error prone. So instead of adding everything and ignoring specific things, we do the opposite of, first ignore everything and then we pick what we actually want. 

I guess this is some kind of ember boilerplate, but are we may be able to come up with something similar?

Reply:
No problem, I don't mind going through the effort of changing it, but the npmignore is generated by the blueprint for Ember addons, which gets updated if the file layout ever changes (which it might soon enough). Every other Ember addon uses the same .npmignore. If any updates to the file structure happen we can simply just run the blueprint command again and use the new npmignore without having to go in to determine which of the files may or may not be used by the addon ecosystem in some way. 

tldr; leaving as is is more maintainable, as the file structure for Ember addons is convention based.